README.md

```diff
@@ -1,44 +1,251 @@
+
+- [Amazon Virtual Private Cloud][vpc].
+- [AWS Identity and Access Management][iam]
+- [Amazon Cloud Watch][acw].
+- [Auto Scaling Groups][asg].
+- [And more].
+
+___
 
 ## Getting Started
 
-TBD
+### How to Install and Configure Zero
+
+There are multiple ways to install zero:
+
+- Install Zero using your systems package manager.
+
+```
+# MacOS
+brew tap commitdev/zero
+brew install zero
+```
+
+- Install Zero by downloading the binary.
+
+Download the latest [zero binary] for your systems archetecture. unzip your download add copy the zero binary to the desired location then add it to your system path.
+
+Zero curretnly supports:
+| System | Support|  Package Manager |
+|---------|:-----:|:------:|
+| MacOS   |  ✅   | `brew` |
+| Linux   |  ✅   |   n/a  |
+| Windows |  ❌   |   n/a  |
+
+### Configure zero dependencies
+
+Zero requires some dependencies to function, run the `zero check` command on your system to find out which other tools/dependencies you might need to install.
+
+![zero-check](./docs/img/zero-check.png)
+
+[AWS CLI], [Kubectl], [Terraform], [jq], [Git], [Wget]
+
+A few caviets:
+
+- For Zero to communicate with your AWS account make sure you [authenticate AWS CLI with your account credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html#cli-configure-files-methods).
+
+  - You can also configure your aws cli during the zero project initilization.
+
+- You need to [register a new domain](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/domain-register.html) / [host a registered domain](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/MigratingDNS.html) you will use to access your infrastructure on [Amazon Route 53](https://aws.amazon.com/route53/).
+
+  - We recommended you have two domains one for staging another for production.
+
+___
+
+## Using zero to spin up your own stack
+
+Using Zero to spin up your infrastructure is easy and straightforward; using a few commands, you can configure and deploy your very own scalable high-performant infrastructure that is production-ready.
+
+A few caveats before getting started:
+
+- It is recomended practice to [create a GitHub org](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/creating-a-new-organization-from-scratch) where your code is going to live and grant [CircleCi Organization access](https://github.com/settings/connections/applications/78a2ba87f071c28e65bb) to your repositories for CI / CD deployment.
+
+- During set-up, you need to create a GitHub personal access token and a CircleCi access token; you should store your generated tokens securely.
+
+- For your CI build to work, you need to opt into the use of third-party orbs you can find this in your CircleCi Org Setting > Security > Allow Uncertified Orbs.
+
+- It takes about 20 - 35 mins for your deployed frontend application to be globally available through AWS CloudFront CDN.
```

Comment:
cloudfront part shouldnt be "A few caveats before getting started:"
maybe do a troubleshoot or FAQ thing at the end?